yml/dp01_dc2.yaml

```diff
@@ -8359,23 +8359,23 @@ tables:
     mysql:datatype: DOUBLE
     tap:column_index: 999
     tap:principal: 0
-    fits:tunit: nmgy*asec^2
+    fits:tunit: nmgy*arcsec^2
```

Comment:
I am not sure why "nmgy" is accepted by the Astropy validator.  It's not in VOUnit or FITS, whether interpreted as a single unit or as nano-"mgy".

The "nanomaggie" is a well-recognized unit in the field from a larger perspective (see, e.g., https://www.sdss3.org/dr8/algorithms/magnitudes.php), so perhaps it was added to Astropy to support this even though it's not in the actual community standards.

Anyway, I am happy to leave well enough along here, especially since it's just the near-deprecated DP0.1.

Reply:
To avoid any ambiguity, I meant "'nanomaggie' is the English name of a well-known informal unit", not "the string 'nanomaggie' is standardized".

There appears to be no community standard for a machine-readable name for this concept.  I am happy to leave it as "nmgy" especially as it appears only in DP0.1.